CodeStarterCamp_Week2/main.swift

```diff
@@ -6,7 +6,40 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
+let myLottoNumbers: [Int] = [4, 31, 11, 7, 28, 45]
+
+var lottoNumbers: Set<Int> = Set<Int>()
+
+func lottoPlease() -> Set<Int>{
+    repeat {
+        let lottoNumber = Int.random(in: 1...45)
+        lottoNumbers.insert(lottoNumber)
+        if lottoNumbers.contains(lottoNumber) {
+        } else {
+            lottoNumbers.insert(lottoNumber)
+        }
+    } while lottoNumbers.count < 6
+    return lottoNumbers
+}
+
+func checkLottoNumbers(numbers: Array<Int>) {
+    let sameNumbers: [Int] = Array(Set(myLottoNumbers).intersection(lottoNumbers))
+    if sameNumbers.count > 0 {
+        print("축하합니다! 겹치는 번호는 ", terminator: "")
+        for value in sameNumbers {
+            if value == sameNumbers[sameNumbers.count-1]{
```

Comment:
띄어쓰기 오타가 있네요 😅
그리고 연산자의 경우 앞 뒤를 띄어써주는게 좋습니다. (사바사 이긴 합니다 ㅎㅎ)
```suggestion
            if value == sameNumbers[sameNumbers.count - 1] {
```